frontend/mr-admin-flate/src/components/redaksjoneltInnhold/AvtaleSchema.ts

```diff
@@ -38,7 +38,9 @@ export const AvtaleSchema = z
       .object({
         startDato: z
           .string({ required_error: "Du må legge inn startdato for avtalen" })
-          .min(10, "Du må legge inn startdato for avtalen"),
+          .min(10, "Du må legge inn startdato for avtalen")
+          .optional()
+          .nullable(),
```

Comment:
I stedet for å gjøre startDato optional, så kan vi kanskje gjøre `opsjonsmodellData.opsjonsmodell` required? 
![image](https://github.com/user-attachments/assets/795d2cc8-2b6d-4c55-9135-cc6f234973f8)